### PR TITLE
LPS-172572 Resize case history columns to emphasize the errors and other table improvements

### PR DESCRIPTION
**Pull request by caiosouza15**

### CODE REVIEW

**Review by rafaelfs98 — APPROVED** [review]
LGTM :fire:

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Cases/CaseResultHistory.tsx [hunk 1]
```diff
@@ -28,85 +31,112 @@ type CaseResultHistoryProps = {
...
+					dateModified: false,
+					issues: false,
+					team: false,
 				},
-				{
-					clickable: true,
-					key: 'build',
-					render: (build) => {
-						return build?.gitHash;
+			}}
```
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Cases/CaseResultHistory.tsx:47]: You need to remove this  to show all columns.

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Table/index.tsx [hunk 2]
```diff
@@ -114,7 +117,7 @@ const Table: React.FC<TableProps> = ({
 
 	return (
 		<>
-			<ClayTable borderless className="testray-table" hover>
+			<ClayTable borderless className="testray-table" hover responsive>
```
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Table/index.tsx:120]: Create a property to validate responsiveness as this change can impact all tables

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Cases/CaseResultHistory.tsx [hunk 3]
```diff
@@ -28,85 +31,112 @@ type CaseResultHistoryProps = {
...
+			resource={testrayCaseResultImpl.resource}
+			tableProps={{
+				columns: [
+					{
+						clickable: true,
+						key: 'dateCreated',
+						render: (date) => (
+							<p style={{maxWidth: '11ch'}}>
+								{' '}
```
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Cases/CaseResultHistory.tsx:60]: You can remove this line.

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Cases/CaseResultHistory.tsx [hunk 4]
```diff
@@ -28,85 +31,112 @@ type CaseResultHistoryProps = {
...
-							>
-								{dueStatus.name}
-							</StatusBadge>
-						);
+					{
+						clickable: true,
+						key: 'build',
+						render: (build) => {
+							return build?.gitHash;
+						},
```
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Cases/CaseResultHistory.tsx:72]: Simplify, returning  directly
```suggestion
						render: (build) => build?.gitHash;,
```

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Cases/CaseResultHistory.tsx [hunk 5]
```diff
@@ -28,85 +31,112 @@ type CaseResultHistoryProps = {
...
+						key: 'dueStatus',
+						render: (dueStatus: PickList) => {
+							return (
+								<StatusBadge
+									type={dueStatus.key as StatusBadgeType}
+								>
+									{dueStatus.name}
+								</StatusBadge>
+							);
+						},
```
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Cases/CaseResultHistory.tsx:104]: Simplify returning directly
```suggestion
						render: (dueStatus: PickList) =>  (
								<StatusBadge
									type={dueStatus.key as StatusBadgeType}
								>
									{dueStatus.name}
								</StatusBadge>
							),
```

**Review by welitonpe — COMMENTED** [review]
Good Job,

**Comment by caiosouza15** [comment]
I need new reviews @welitonpe @rafaelfs98

**Comment by caiosouza15** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:forward

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Table/index.tsx [hunk 6]
```diff
@@ -160,6 +170,7 @@ const Table: React.FC<TableProps> = ({
 								contextMenuState.visible
 							}
 							className={classNames('table-row', {
+								'hightligth-bar': highlight && highlight(item),
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Table/index.tsx:173]: highlight-bar

@welitonpe

**Pull request closed without merging**